wordpress-mobile/WordPress-Android
# Placing quick start focus point over related views

Part of #7707

**Based on #8015 (so I can chain PR's) so it must be merged first.**
**Feel free to remove "Not Ready for Review" when it's done.**

This PR adds an animated focus point view over related My Site rows and bottom nav buttons.

To test:

1. Navigate to Quick Start activity from My Site fragment.
2. Click on each tutorial, and make sure that the Quick Start focus point appears at the right place.

[![Image from Gyazo](https://i.gyazo.com/455a0412d6a0a7fb5d28f793db53551a.png)](https://gyazo.com/455a0412d6a0a7fb5d28f793db53551a)
